### PR TITLE
Bugfix/bug 798106

### PR DESCRIPTION
**Pull request by baylp2022**
BUG-798106 - Unable to set speculative exec policy props and custom retry count in pega-helm-chart

### CODE REVIEW

File: charts/pega/templates/pega-environment-config.yaml [hunk 1]
```diff
@@ -110,33 +110,36 @@ data:
...
-  # Specify the number of retries when custom retry policy is enabled.
+  # Specify the number of retry attempts when `customRetryPolicyEnabled` is true. For Pega Platform '23 and earlier
+  # releases use the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount.
   CASSANDRA_SPECULATIVE_EXECUTION_POLICY: "{{ .Values.dds.speculativeExecutionPolicy }}"
-  # Enable the speculative execution policy for retrieving data from your Cassandra service. When enabled, the Pega
-  # Platform will send a query to multiple nodes in your Cassandra service and process the first query response. This
-  # provides lower perceived latencies for your deployment, but puts greater load on your Cassandra service.
-  # This property is for use with Pega Infinity '23 and lower.
+  # Enable the speculative execution policy for retrieving data from your Cassandra service for Pega Platform '23 and
+  # earlier releases. When enabled, the Pega Platform will send a query to multiple nodes in your Cassandra service and
```
Comment (taz-pega-work) [inline charts/pega/templates/pega-environment-config.yaml:126]: cleanup a little wording>>
When enabled, the Pega Platform will send a query to multiple nodes in your Cassandra service and process the first query response.
to
When enabled, Pega Platform sends a query to multiple nodes in your Cassandra service and processes the first response.

**Review by taz-pega-work — APPROVED** [review]
one tweak would help clarify the action...but up to you.

**Review by taz-pega-work — APPROVED** [review]
those file changes look right on! thanks for tweaking these details!

**Review by kingakowalska1 — APPROVED** [review]
Looks great, thanks!

**Comment by taz-pega-work** [comment]
with the content fixed and approved, merging this PR

**Pull request merged**